package.json

```diff
@@ -94,8 +90,6 @@
     "@types/supertest": "^2.0.8",
     "@typescript-eslint/eslint-plugin": "^4.14.0",
     "@typescript-eslint/parser": "^4.14.0",
-    "aws-cdk": "^1.22.0",
-    "aws-sdk": "^2.783.0",
```

Comment:
Isn't this used for fetching s3 config? https://github.com/guardian/support-dotcom-components/blob/main/src/utils/S3.ts

Reply:
This dependency is already in the `dependencies` block

Reply:
Ahh gotcha, nice! 👍